.github/workflows/release.yml

```diff
@@ -12,13 +12,11 @@ name: Release
 env:
   RUST_TOOLCHAIN: nightly-2023-02-26
 
-  # FIXME(zyy17): Would be better to use `gh release list -L 1 | cut -f 3` to get the latest release version tag, but for a long time, we will stay at 'v0.1.0-alpha-*'.
-  SCHEDULED_BUILD_VERSION_PREFIX: v0.1.0-alpha
+  SCHEDULED_BUILD_VERSION_PREFIX: v0.2.0
 
-  # In the future, we can change SCHEDULED_PERIOD to nightly.
-  SCHEDULED_PERIOD: weekly
+  SCHEDULED_PERIOD: nightly
 
-  CARGO_PROFILE: weekly
+  CARGO_PROFILE: nightly
```

Comment:
we don't have this profile. This field should be `weekly` like before

Reply:
@waynexia OK, I will modify `Cargo.toml`.